projects/organization-management/src/app/facades/organization-management.facade.ts

```diff
@@ -15,4 +27,29 @@ export class OrganizationManagementFacade {
   usersError$ = this.store.pipe(select(getUsersError));
   usersLoading$ = this.store.pipe(select(getUsersLoading));
   currentUser$ = this.store.pipe(select(getCurrentUser));
+  customer$ = this.store.pipe(select(getLoggedInCustomer));
+
+  addUser(user: User) {
+    this.store.dispatch(
+      new AddUser({
+        user,
+        successMessage: 'account.profile.update_profile.message',
+        successRouterLink: '/account/organization/users',
```

Comment:
I think, this routing should be triggered in an effect on "AddUserSuccess", so it is not needed as a payload here. Also: use relative links, otherwise the user-management doesn't work as a standalone application.

Reply:
Hey Danilo!! Long time no see :) I was inspired by the account facade. Did it this way to be inline with what you had before, but if you think it can be improved I will gladly do it ;)

Reply:
I found a solution 😉 👍